src/makeCalculator.js

```diff
@@ -37,7 +37,47 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add: function(num) {
+      this.result += num;
+
+      return this;
+    },
+
+    subtract: function(num) {
+      this.result -= num;
+
+      return this;
+    },
+
+    multiply: function(num) {
+      this.result *= num;
+
+      return this;
+    },
+
+    divide: function(num) {
+      if (num !== 0) {
```

Comment:
In the divide function, there's no handling for the case when the number is 0. This will cause a division by zero error. Please handle this case properly, perhaps by throwing an error.